src/build.d

```diff
@@ -401,12 +401,14 @@ alias backend = makeRuleWithArgs!((MethodInitializer!BuildRule builder, BuildRul
 /// Returns: the rules that generate required string files: VERSION and SYSCONFDIR.imp
 alias versionFile = makeRule!((builder, rule) {
     alias contents = memoize!(() {
+        /*
         if (dmdRepo.buildPath(".git").exists)
         {
             auto gitResult = tryRun([env["GIT"], "describe", "--dirty"]);
```

Comment:
This currently returns something like `v2.098.0-526-gdc8325b16` and is at least used by CircleCI. This overrides the `VERSION` file, and makes the new compiler's `__VERSION__` remain stuck at `2098`, so that the required change in `TemplateInstanceBox.opEquals` isn't applied. :/

Reply:
Solved by https://github.com/dlang/dmd/pull/13593#issuecomment-1026801838.